examples/gallery/histograms/histogram.py

```diff
@@ -0,0 +1,34 @@
+"""
+Histogram
+---------
+The :meth:`pygmt.Figure.histogram` method can plot regular histograms.
+"""
+
+import numpy as np
+import pygmt
+
+np.random.seed(100)
+
+# generate example topography data
+mu = 100  # mean of distribution
+sigma = 25  # standard deviation of distribution
+data = mu + sigma * np.random.randn(521)
```

Comment:
Perhaps use English instead of Greek :wink:

```suggestion
mean = 100  # mean of distribution
stddev = 25  # standard deviation of distribution
data = mean + stddev * np.random.randn(521)
```

Reply:
Good point 😉.